states/minnesota/elections.toml

```diff
@@ -8,7 +8,85 @@ date = 2020-08-11
 name = "Congresssional Primary"
 original_date = 2020-08-11
 
+[20200811.registration]
+online_by = 2020-07-21T23:59:00
+received_by = 2020-07-21T17:00:00
+in_person_by = 2020-08-11T20:00:00
+sources = [
+    "https://www.sos.state.mn.us/elections-voting/election-day-voting/",
+    "https://www.sos.state.mn.us/elections-voting/election-day-voting/voting-hours/",
+    "https://www.sos.state.mn.us/elections-voting/register-to-vote/registration-faqs/"
+]
+
+[20200811.absentee.application]
+online_by = 2020-08-10 # Legally this is true, but you'll never receive it.
+received_by = 2020-08-10 # Legally this is true, but you'll never receive it.
+fax_by =  2020-08-10 # Legally this is true, but you'll never receive it.
+email_by =  2020-08-10 # Legally this is true, but you'll never receive it.
```

Comment:
I think I'd do two or three weeks on this. 1 to send in, 1 to get ballot and 1 to send ballot. Only received_by will be reminded a week out so maybe ~2 for all and received_by will be offset further.